### ISSUE TITLE
cabal install pangraph fails

### ISSUE THREAD

**Issue by zouroboros** [thisiswhereitype/gmlp#24]
cabal install pangraph fails with the following error message:
Failed to install pangraph-0.1.1.5
Build log ( /home/zouroboros/Seafile/Studium/Computational Science/Complex Systems/graph-partioning/.cabal-sandbox/logs/ghc-8.2.2/pangraph-0.1.1.5-8gnv6s7siSqCW67qtywlss.log ):
cabal: Entering directory '/tmp/cabal-tmp-18188/pangraph-0.1.1.5'
Configuring pangraph-0.1.1.5...
Preprocessing library for pangraph-0.1.1.5..
Building library for pangraph-0.1.1.5..
[1 of 9] Compiling Pangraph         ( src/Pangraph.hs, dist/dist-sandbox-5bd21440/build/Pangraph.o )

src/Pangraph.hs:68:17: error:
    Not in scope: ‘Alga.graph’
    Perhaps you meant one of these:
      ‘Alga.toGraph’ (imported from Algebra.Graph.Class),
      ‘Alga.path’ (imported from Algebra.Graph.Class)
    Module ‘Algebra.Graph.Class’ does not export ‘graph’.
   |
68 |     toGraph p = Alga.graph (vertexList p) (map edgeEndpoints $ edgeList p)
   |                 ^^^^^^^^^^
cabal: Leaving directory '/tmp/cabal-tmp-18188/pangraph-0.1.1.5'
cabal: Error: some packages failed to install:
pangraph-0.1.1.5-8gnv6s7siSqCW67qtywlss failed during the building phase. The
exception was:
ExitFailure 1

**Comment by snowleopard** [comment]
@zouroboros Thank you for reporting! This should be fixed in the above PR.

@thisiswhereitype Can you release a new version once the PR is merged?

**Comment by thisiswhereitype** [comment]
@zouroboros Thanks also for taking the time.
@snowleopard Unless I have misunderstood the https://pvp.haskell.org/ this suffices a major release as there has been a change to an exported function?

**Comment by snowleopard** [comment]
@thisiswhereitype I didn't change the exported function -- it behaves exactly as before, but now it compiles with more versions of Alga. So, I don't think this should be a major release.

**Comment by thisiswhereitype** [comment]
@zouroboros A new minor version is released on hackage. Thanks
